### PR TITLE
PSI-212 : Add README.md for VAT replacement

### PR DESCRIPTION
**Pull request by leungkinghin-ct**
Documentation for VAT replacement scripts

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,54 @@
 # commercetools-cli-scripts
 Collection of node.js based scripts and utils for CTP resource management
+
+## Requirements
+Download Node.js installer
```
Comment (ahmetoz) [inline README.md]:
```suggestion

- Node.js version 12.
- Npm version 6.

Install nodejs into your machine to be able to run the scripts, follow the installation link for instructions: https://nodejs.org/en/download/

```

File: README.md [hunk 2]
```diff
@@ -1,2 +1,54 @@
 # commercetools-cli-scripts
 Collection of node.js based scripts and utils for CTP resource management
```
Comment (ahmetoz) [inline README.md]:
```suggestion
The repository consist nodejs based CLI scripts to update resources in commercetools.
```

File: README.md [hunk 3]
```diff
@@ -1,2 +1,54 @@
...
+
+## Requirements
+Download Node.js installer
+https://nodejs.org/en/download/
+
+## Setup
+Inside /features/shared folder, input following command in terminal to install required javascript library
```
Comment (ahmetoz) [inline README.md]: Here you need to give one command to install all you don't need to specify every dependency. Less description is more ready to go commands. Following the current documentation is hard. This is why I said before using one package.json.

**Review by ahmetoz — COMMENTED** [review]
In general, it's hard to follow, can we make it shorter?

File: README.md (hunk 3 above)
Comment (leungkinghin-ct) [reply README.md]: https://github.com/commercetools/commercetools-cli-scripts/commit/d7e9a3b2f9074b75fbb082811ccd724c264a610b

File: README.md (hunk 2 above)
Comment (leungkinghin-ct) [reply README.md]: https://github.com/commercetools/commercetools-cli-scripts/commit/d7e9a3b2f9074b75fbb082811ccd724c264a610b

File: README.md (hunk 1 above)
Comment (leungkinghin-ct) [reply README.md]: https://github.com/commercetools/commercetools-cli-scripts/commit/d7e9a3b2f9074b75fbb082811ccd724c264a610b

**Review by ahmetoz — APPROVED** [review]
